lesson13/js/main.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
+//Modal
+const modalContainer = document.getElementById('js-modalContainer');
+const modalBg = document.createElement('div');
+const modalWindow = document.createElement('div');
+
+function createModalWindow() {
+	const modal = document.createDocumentFragment();
+	const createMenuBtn = document.createElement('button');
+	const createCloseBtn = document.createElement('button');
+	const createMenuContainer = document.createElement('div');
+
+	//modal container
+	modalWindow.classList.add('bl_modal');
+	modalWindow.setAttribute('id', 'js-modalContainer');
+
+	//menu button
+	createMenuBtn.setAttribute('id', 'js-loadmenuBtn');
+	createMenuBtn.setAttribute('class', 'btn');
+	createMenuBtn.textContent = `MENU`;
+
+	//close button
+	createCloseBtn.setAttribute('id', 'js-closeModal');
+	createCloseBtn.setAttribute('class', 'el_closeModalBtn');
+	createCloseBtn.textContent = '×';
+
+	createMenuContainer.setAttribute('id', 'js-menu_container');
+
+	modal.appendChild(createMenuBtn);
+	modal.appendChild(createCloseBtn);
+	modal.appendChild(createMenuContainer);
+	modalWindow.appendChild(modal);
+	document.body.appendChild(modalWindow);
+
+	const clickBtn = document.getElementById('js-loadmenuBtn');
+	clickBtn.addEventListener(
+		'click',
+		(event) => {
+			callMenuContents();
+			event.stopPropagation();
+		},
+		false
+	);
```

Comment:
This can be cut out as a function.

Reply:
Thank you for review!

I realized that if I didn't reassign the constants, I would multiply them!
I was able to make the button that calls the menu a function!

[6a49c07](https://github.com/nyattsu72/hanson-frontend-cource/pull/18/commits/6a49c07a45c2d5dcbd80b4315224d2a5a9cd4891)